mem0/graphs/utils.py

```diff
@@ -54,6 +54,52 @@
 
 Adhere strictly to these guidelines to ensure high-quality knowledge graph extraction."""
 
+FALKORDB_QUERY = """
```

Comment:
What's this query used for?
It seems to me like this query is a bit wasteful, we can improve it, please suggest Mem0 team a quick query optimization session.